CMakeLists.txt

```diff
@@ -184,8 +186,10 @@ elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|amd64|AMD64")
 elseif(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64")
 	set(_M_ARM 1)
 	set(_M_ARM_64 1)
-	add_definitions(-D_M_ARM=1 -D_M_ARM_64=1)
-	add_definitions(-march=armv8-a+crc)
+	add_definitions(-D_M_ARM=1)
+	add_definitions(-D_M_ARM_64=1)
+	# CRC instruction set is used in the CRC32 hash function
+	check_and_add_flag(HAVE_ARCH_ARMV8 -march=armv8-a+crc)
```

Comment:
Did we figure out what'll happen if Dolphin is built without the CRC extensions?